### PR TITLE
feat: include installation of font

### LINKED ISSUE #774: improve loading of map thumbnail images
**Please describe the feature**

As a tech collaborator, I want to have load map thumbnails faster and with correct font rendering, so that I can show correct thumbnails without waiting for too long.

**Describe the solution you would like**
There are two problems:

1. For larger maps, the loading time can take a really long time (over one minute).
2. Fonts are not being rendered, it just shows rectangles.

A suggested solution is to pre-process all of the images from SVG into PNG format. Perhaps this is more suitable to be done in the [data-generation repo](https://github.com/MetabolicAtlas/data-generation). This should solve both of the problems, especially if the PNG files are resized into smaller ones (max 2000px wide for example), since it offloads the preprocessing time to the build-step and the text in PNG format would not cause any problems.

If we want to go with other solutions that takes each problem individually, it might make more sense to create two new issues to replace this one.

**Additional context**
[This comment](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/753#issuecomment-964079642), as well as discussion in the related issue, includes examples of the problems.

### PR DESCRIPTION
**Pull request by e0**
**Related issue(s) and PR(s)**  
This PR closes #774.

**Type of change**  

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
 
**List of changes made**  

- Install a default font (`ttf-dejavu`) alongside `fontconfig`. (Problem before was that no fonts were installed.)
- Install other build tools to address `Pango-CRITICAL` errors.
- Update `sharp` version.

**Screenshot of the fix**  

image

**Testing**  

- A rebuild is needed due to new dependencies
- Relative urls that can be reused both for production and local testing
- Instructions on how to test
1. Rebuild the `api` container: `docker-compose --env-file env-local.env -f docker-compose.yml -f docker-compose-local.yml up -d --no-deps --build api`.
2. Visit a thumbnail endpoint, for example: `/api/v2/svg-thumbnail/mitochondria?model=Human-GEM&height=300` or `/api/v2/svg-thumbnail/acylglycerides_metabolism?model=Human-GEM&height=300`.
3. Verify that it does not take too long to load and that the text loads instead only showing rectangles.

**Further comments**  

We have [previously documented](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/753#issuecomment-964079642) that the thumbnail takes a really long time to generate. When I tested now, the same endpoint (`/api/v2/svg-thumbnail/mitochondria?model=Human-GEM&height=300`) takes under `2s` for me. So the speed seems to be no longer a problem. Please let me know if this still takes too long for you though.

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [x] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by inghylt — APPROVED** [review]
Works well for me!

File: api/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM node:16-alpine
-RUN apk add --no-cache fontconfig
+
+RUN apk add --no-cache --virtual .gyp python3 make g++
+RUN apk --no-cache --update add fontconfig ttf-dejavu
```
Comment (MalinAhlberg) [inline api/Dockerfile:4]: just curious, why is `--virtual` and `--update` needed at these lines?

**Review by MalinAhlberg — APPROVED** [review]
Works well! And is very fast, good job!

File: api/Dockerfile (hunk 1 above)
Comment (e0) [reply api/Dockerfile:4]: Sorry but I didn't look enough into this. I sampled these from a few different sources for people who had problems with `pango` (a dependency for `fontconfig`) and missing fonts. I imagine we don't need either here but I saw a few posts similar to this. Hope it's just some best practices that I don't fully understand.

**Review by nanjiangshu — APPROVED** [review]
Awesome!

**Pull request merged**